src/components/application-loader/initializers/index.ts

```diff
@@ -12,7 +12,11 @@ import { fetchFrontendConfig } from './fetch-frontend-config'
 import { loadDarkMode } from './load-dark-mode'
 
 const customDelay: () => Promise<void> = async () => {
-  if (window.localStorage.getItem('customDelay')) {
+  if (
+    typeof window !== 'undefined' &&
+    typeof window.localStorage !== 'undefined' &&
+    (window.location.search.startsWith('?customDelay=') || window.localStorage.getItem('customDelay'))
+  ) {
     return new Promise((resolve) => setTimeout(resolve, 5000))
```

Comment:
Do we want to use the value of the local storage key / the query parameter as the delay to allow other values than 5 seconds?

Reply:
As you can see: no.

Reply:
Why not?

Reply:
It's only a test flag.

Reply:
I want to enjoy my freetime watching the animation and not pressing the reload button every 5 seconds...

Reply:
> It's only a test flag.

Okay then, if you insist on it...